app/Views/admin/categories/Categories.php

```diff
@@ -1 +1,194 @@
+                                                <td>Imagen.jpeg</td>
+                                                <td>
+                                                    <button type="button" class="btn btn-primary btn-icon waves-effect waves-light"><i class="ri-edit-2-fill"></i></button>
+                                                    <button type="button" class="btn btn-danger btn-icon waves-effect waves-light"><i class="ri-delete-bin-5-line"></i></button>
+                                                </td>
+                                            </tr>
+                                            <tr>
+                                                <td>06</td>
+                                                <td>
+                                                    <div class="d-flex align-items-center fw-medium">
+                                                        <a href="javascript:void(0);">Serie L</a>
+                                                    </div>
+                                                </td>
+                                                <td>Icono.jpeg</td>
+                                                <td>Imagen.jpeg</td>
+                                                <td>
+                                                    <button type="button" class="btn btn-primary btn-icon waves-effect waves-light"><i class="ri-edit-2-fill"></i></button>
+                                                    <button type="button" class="btn btn-danger btn-icon waves-effect waves-light"><i class="ri-delete-bin-5-line"></i></button>
+                                                </td>
+                                            </tr>
+                                            <tr>
+                                                <td>07</td>
+                                                <td>
+                                                    <div class="d-flex align-items-center fw-medium">
+                                                        <a href="javascript:void(0);">Serie I</a>
+                                                    </div>
+                                                </td>
+                                                <td>Icono.jpeg</td>
+                                                <td>Imagen.jpeg</td>
+                                                <td>
+                                                    <button type="button" class="btn btn-primary btn-icon waves-effect waves-light"><i class="ri-edit-2-fill"></i></button>
+                                                    <button type="button" class="btn btn-danger btn-icon waves-effect waves-light"><i class="ri-delete-bin-5-line"></i></button>
+                                                </td>
+                                            </tr>
+                                        </tbody>
+                                    </table>
```

Comment:
Puedes añadir un tfoot porfavor @FernandoJoachin 

```suggestion
                                            </tr>
                                        </tbody>
                                        <tfoot>
                                            <tr>
                                               <th>ID</th>
                                               <th>Nombre</th>
                                               <th>Icono</th>
                                               <th>Imagen</th>
                                               <th>Acciones</th>
                                              </tr>
                                          </tfoot>
                                    </table>
```